configs/dask_cluster/dev_kube_cluster/scheduler.yml

```diff
@@ -1,14 +1,14 @@
 kind: Pod
 spec:
-  affinity:
-    nodeAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        nodeSelectorTerms:
-        - matchExpressions:
-          - key: network
-            operator: In
-            values:
-            - 10gb
+  # affinity:
```

Comment:
Maybe just delete these unless someone notices the scheduler really has to be on a fast network?